### PR TITLE
feat: card with icons component

### PR DESCRIPTION
**Pull request by Kevinemug**
done with card with icon component

### CODE REVIEW

File: src/Components/cardWithIcon.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+<div class="grid grid-cols-3 gap-[80px] px-[86px]">
+  
+    <div class="flex gap-[24px] items-center">
+      <div>
+        <img src="/src/Assets/media/Hand-coins-icon.svg" alt="">
+      </div>
+      <div>
+      <h5 class="text-primary font-montBold text-[20px]">Kostengünstig</h5>
+    <p class="text-white font-montRegular text-[16px]"> niedriges Lohnkostenniveau im europäischen Vergleich</p>
```
Comment (Angekarara) [inline src/Components/cardWithIcon.html]: the text has to be 20 px not 16px

File: src/Components/cardWithIcon.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    
+     </div>
+    </div>
+    <div class="flex gap-[24px] items-center">
+      <div>
+        <img src="/src/Assets/media/Hand-coins-icon.svg" alt="">
+      </div>
+      <div>
+      <h5 class="text-primary font-montBold text-[20px]">Kostengünstig</h5>
+    <p class="text-white font-montRegular text-[16px]"> niedriges Lohnkostenniveau im europäischen Vergleich</p>
```
Comment (Angekarara) [inline src/Components/cardWithIcon.html]: this text has to be 20px too

File: src/Components/cardWithIcon.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
```
Comment (Angekarara) [inline src/Components/cardWithIcon.html]: this text has to be 20px too

**Review by Angekarara — CHANGES_REQUESTED** [review]
those three paragraphs on each icon has to be 20px not 16px

**Review by HIRWA13 — APPROVED** [review]
Hi team, the changes you made matches the ones that we requested now I see that the design is good you can merge!

**Pull request merged**